client/main.lua

```diff
@@ -87,6 +123,7 @@ end)
 -- Callbacks
 
 RegisterNUICallback("NUIFocusOff", function()
+	InATM = false
```

Comment:
```suggestion
    InATM = false
```
Incorrect linting / spacing